spec/features/admin_deletes_program_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'admin deletes a program', %{
+  As an admin
+  I want to be able to delete a program
+} do
+
+  # [ ] Admins are able to verify their credentials
```

Comment:
Split the admin can log in tests into their own spec. It's not really related to the program deletion.